src/sentry/static/sentry/app/components/group/sidebar.tsx

```diff
@@ -134,20 +139,9 @@ class GroupSidebar extends React.Component<Props, State> {
         },
       },
       {
-        complete: () => {
-          api.request(`/issues/${group.id}/participants/`, {
-            success: data => {
-              this.setState({
-                participants: data,
-                error: false,
-              });
-              clearIndicators();
-            },
-            error: () => {
-              this.setState({error: true});
-              clearIndicators();
-            },
-          });
+        complete: async () => {
+          await this.fetchParticipants();
+          clearIndicators();
```

Comment:
Do you need a `catch` here to clear indicators?

Reply:
@markstory Nope, there's a try/catch in `fetchParticipants`, so this should never reject.